src/memkind_memtier.c

```diff
@@ -1105,16 +1105,14 @@ MEMKIND_EXPORT void *memtier_realloc(struct memtier_memory *memory, void *ptr,
 MEMKIND_EXPORT void *memtier_kind_realloc(memkind_t kind, void *ptr,
                                           size_t size)
 {
-    size_t old_size = 0u;
     if (size == 0 && ptr != NULL) {
-        old_size = jemk_malloc_usable_size(ptr);
 #ifdef MEMKIND_DECORATION_ENABLED
         if (memtier_kind_free_pre)
             memtier_kind_free_pre(&ptr);
 #endif
-
-        if (pol == MEMTIER_POLICY_DATA_HOTNESS) {
-//             unregister_block(ptr);
+    size_t old_size = jemk_malloc_usable_size(ptr);
```

Comment:
Incorrect indentation
```suggestion
        size_t old_size = jemk_malloc_usable_size(ptr);
```